requre/helpers/files.py

```diff
@@ -91,11 +91,48 @@ def read_file_content(
         return content
 
     @classmethod
-    def _copy_logic(cls, cassette: Cassette, pathname: str, keys: list) -> None:
+    def __read_file(cls, content, pathname):
+        with BytesIO(content) as fileobj:
+            with tarfile.open(
+                mode=f"r:{cls.tar_compression}", fileobj=fileobj
+            ) as tar_store:
+                tarinfo_1st_member = tar_store.getmembers()[0]
+                if tarinfo_1st_member.isfile():
+                    with open(pathname, mode="wb") as output_file:
+                        output_file.write(
+                            tar_store.extractfile(tarinfo_1st_member).read()
+                        )
```

Comment:
```suggestion
                    with open(pathname, mode="wb") as output_file:
                        output_file.write(
                            tar_store.extractfile(tarinfo_1st_member).read()
                        )
                        return
```
So we can get rid of the indentation?

Reply:
not sure. have to look into code

Reply:
seems possible. but fro my perspective it is less readable :-)